assignment1q1/Assignment1q1.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */

```

Comment:
It is better to either modify such autogenerated comments so that they contain some useful information or to remove them. Otherwise they only introduce unnecessary noise to the code.